### PR TITLE
Update pourbaix heatmap plots

### PR DESCRIPTION
**Pull request by JoshuaNgXY**
## Summary
- Removed x, y, and z labels on hovertext as they are redundant.
- Changed the color of the ions to be a little more off-white.
- Updated the heatmap scale to display ΔG<sub>pbx</sub> beyond 1 ev/atom.
- Updated the color scale to favor lower energies.
- Updated the color of the lines on the heatmap so that they are visible when the heatmap is turned off.

### Before
image

### After
image

### Before
image
image

### After
image
image

### CODE REVIEW

**Comment by JoshuaNgXY** [comment]
Failing in a part of code that was not edited in this PR.

File: crystal_toolkit/components/pourbaix.py [hunk 1]
```diff
@@ -464,13 +465,13 @@ def get_text_size(available_vertical_space):
...
+                hoverlabel.append(hovertext)
+            hoverlabel = np.reshape(hoverlabel, list(decomposition_e.shape))
 
             # Enforce decomposition limit energy
-            decomposition_e = np.min(
-                [decomposition_e, np.ones(decomposition_e.shape)], axis=0
-            )
+            # decomposition_e = np.min(
+            #     [decomposition_e, np.ones(decomposition_e.shape)], axis=0
+            # )
```
Comment (JoshuaNgXY) [inline crystal_toolkit/components/pourbaix.py:474]: This must be turned off so that the Gibbs Energy is not capped.

**Comment by JoshuaNgXY** [comment]
Failing in a part of code not edited in this PR. It seems that this failure is common to the repo as other PRs and commits have the same error messages.

**Comment by tschaume** [comment]
@JoshuaNgXY @yang-ruoxi I've fixed the issues with the tests. Feel free to merge and release when ready.

**Pull request merged**